bc/core/utils/status/templates.py

```diff
@@ -104,3 +109,41 @@
 
 #CL{docket_id}""",
 )
+
+
+BLUESKY_FOLLOW_A_NEW_CASE = BlueskyTemplate(
+    link_placeholders=["docket_link"],
+    str_template="""I'm now following {docket}:
+
+[Docket]({docket_link})
+
+#CL{docket_id}""",
+)
+
+BLUESKY_FOLLOW_A_NEW_CASE_W_ARTICLE = BlueskyTemplate(
+    link_placeholders=["docket_link", "article_url"],
+    str_template="""I'm now following {docket}:
+
+[Docket]({docket_link}) | [Article Link]({article_url})
```

Comment:
Staring at this a bit, I think this might be better (and worth tweaks on the other templates too):

```suggestion
[Case Link]({docket_link}) | [Background Info]({article_url})
```